lib/config/migration.ts

```diff
@@ -195,6 +195,9 @@ export function migrateConfig(
         } else if (val === false) {
           migratedConfig.trustLevel = 'low';
         }
+      } else if (key === 'ignoreNpmrcFile') {
```

Comment:
This won't catch CLI or env.. so maybe we treat it as breaking instead?

Reply:
Or add some small migration code for cli and env?